src/mb_enhanced_cover_art_uploads/providers/spotify.ts

```diff
@@ -11,4 +12,14 @@ export class SpotifyProvider extends HeadMetaPropertyProvider {
     protected override is404Page(document_: Document): boolean {
         return qsMaybe('head > meta[property="og:title"]', document_) === null;
     }
+
+    protected override fetchPage(url: URL, options?: RequestOptions): Promise<string> {
+        return super.fetchPage(url, {
+            ...options,
+            headers: {
+                ...options?.headers,
+                'User-Agent': null,
```

Comment:
Does it also work when you provide an empty string? Otherwise we need to suppress the type error raised by TypeScript somehow.